src/mumble/Plugins.cpp

```diff
@@ -632,10 +623,10 @@ void Plugins::fetchedUpdatePAPlugins(QByteArray data, QUrl) {
 		n = n.nextSibling();
 	}
 
-	QDir qd(qsSystemPlugins, QString(), QDir::Name, QDir::Files | QDir::Readable);
-	QDir qdu(qsUserPlugins, QString(), QDir::Name, QDir::Files | QDir::Readable);
+	QDir systemDir(loadPaths[0], QString(), QDir::Name, QDir::Files | QDir::Readable);
+	QDir downloadDir(downloadPath, QString(), QDir::Name, QDir::Files | QDir::Readable);
```

Comment:
The system directory is the main one, for example `plugins` inside the installation directory on Windows.

The download directory is the one in the user's home folder, for example `%AppData%\Mumble\Plugins`.

Reply:
But why do we even need the system dir when seemingly all we want to do is to store downloaded plugins into the download dir?

Besides: Relying on an implicit ordering inside `loadPaths` (so that the first entry is the system dir) seems rather error prone.

Reply:
The code currently checks whether the plugins provided by Mumble match (SHA-1) the ones provided by our server.

Instead of just checking the shared libraries in the first directory, we could iterate through all of them.

Reply:
In order to know which ones shall be updated?

In that case it doesn't matter all that much what we end up doing here, since the new plugin framework comes with auto-update functionality for plugins anyways. I expect that feature to deprecate the approach currently implemented.

Reply:
In that case, do we perhaps want to put this PR on hold?

Reply:
Given that our current plan is to not release 1.4.0 without the plugin framework, that could make sense, yes :thinking: 
I'll just have to remember making the appropriate changes in the plugin PR though ^^

Reply:
Could you check that I didn't forget anything in https://github.com/mumble-voip/mumble/pull/3743/commits/140e7465924ae88c0b589d060626ed45df47ff7b ?

Reply:
Looks perfect to me.

Reply:
Okay then in that case I think we can close this PR. What do you think?

Reply:
Yep, closed.